cdk/lib/cognito-passwordless.ts

```diff
@@ -639,20 +792,222 @@ export class Passwordless extends Construct {
+        requestModels: {
+          "application/json": completeRegistrationModel,
+        },
+      });
+
+      // authenticators/list
+      listResource.addCorsPreflight(defaultCorsOptionsWithAuth);
+      listResource.addMethod("POST", undefined, {
+        authorizer: authorizer,
+        requestParameters: {
+          "method.request.querystring.rpId": true,
+        },
+        requestValidator,
+      });
+
+      // authenticators/delete
+      const deleteCredentialsModel = new cdk.aws_apigateway.Model(
+        scope,
+        `DeleteCredentialModel${id}`,
+        {
+          restApi: this.fido2Api,
+          contentType: "application/json",
+          description: "Delete FIDO2 credential request body",
+          modelName: "credentialDelete",
+          schema: {
+            type: cdk.aws_apigateway.JsonSchemaType.OBJECT,
+            required: ["credentialId"],
+            properties: {
+              credentialId: {
+                type: cdk.aws_apigateway.JsonSchemaType.STRING,
+                minLength: 1,
+              },
+            },
+          },
+        }
+      );
+      deleteResource.addCorsPreflight(defaultCorsOptionsWithAuth);
+      deleteResource.addMethod("POST", undefined, {
+        authorizer: authorizer,
+        requestValidator,
+        requestModels: {
+          "application/json": deleteCredentialsModel,
+        },
+      });
+
+      // register-authenticator/update
+      const updateCredentialsModel = new cdk.aws_apigateway.Model(
+        scope,
+        `UpdateCredentialModel${id}`,
+        {
+          restApi: this.fido2Api,
+          contentType: "application/json",
+          description: "Update FIDO2 credential request body",
+          modelName: "credentialUpdate",
+          schema: {
+            type: cdk.aws_apigateway.JsonSchemaType.OBJECT,
+            required: ["credentialId", "friendlyName"],
+            properties: {
+              credentialId: {
+                type: cdk.aws_apigateway.JsonSchemaType.STRING,
+                minLength: 1,
+              },
+              friendlyName: {
+                type: cdk.aws_apigateway.JsonSchemaType.STRING,
+                minLength: 1,
+                maxLength: 256,
+              },
+            },
+          },
+        }
+      );
+      updateResource.addCorsPreflight(defaultCorsOptionsWithAuth);
+      updateResource.addMethod("POST", undefined, {
+        authorizer: authorizer,
+        requestValidator,
+        requestModels: {
+          "application/json": updateCredentialsModel,
+        },
+      });
+
+      // sign-in-challenge
+      const signInChallenge =
+        this.fido2Api.root.addResource("sign-in-challenge");
+      signInChallenge.addCorsPreflight(defaultCorsOptionsWithoutAuth);
+      signInChallenge.addMethod(
+        "POST",
+        new cdk.aws_apigateway.LambdaIntegration(this.fido2challengeFn),
+        {
+          authorizer: undefined, // public API
+        }
```

Comment:
Why explicitly indicate this? isn't that the default?

Reply:
Just wanted to be super explicit about it in the source, because it's so important